content/en/docs/doc-contributor-tools/linkchecker/README.md

```diff
@@ -0,0 +1,66 @@
+# Internal link checking tool
+
+You can use the https://github.com/wjdp/htmltest tool to check for broken links on **english content**. This is a helpful when refactoring sections of content, moving pages around, or renaming files or page headers.
+
+## What is and isn't checked
+
+The link checker scans generated HTML files, **not** raw markdown. You can see the full configuration in the `.htmltest.yml` file in the root of `kubernetes/website`.
+
+The link checker scans the following:
+
+- All content generated from markdown in the `/content/en/docs` directory **excluding**:
+  - Generated API references
+- All internal links **excluding**:
+  - Empty hashes (`<a href="#">` or `[title](#)`) and empty hrefs (`<a href="">` or `[title]()`)
+  - Internal links to images and other media files
+
+The link checker **does not** scan the following:
+
+- Links included in the site's "chrome": top and side menu links, footer links, links in a page's `<head>` section such as those to CSS stylesheets, scripts, and meta information
+- Top level pages _and_ their children, for example: `/training`, `/community`, `/case-studies/adidas`
```

Comment:
I'm squinting hard at those italics, but I leave it to your discretion.

![buffy](https://user-images.githubusercontent.com/3210446/81775482-c5f47800-94a1-11ea-8bbe-773e50ea7557.gif)